src/main.cpp

```diff
@@ -89,11 +93,21 @@ int main(int argc, char *argv[]) {
       }
       break;
     case 'V':
+      use_bus_addr = false;
       sscanf(optarg, "%x", &ftdi_vid);
       break;
     case 'p':
+      use_bus_addr = false;
       sscanf(optarg, "%x", &ftdi_pid);
       break;
+    case 'B':
```

Comment:
Maybe we can emit some error/warning messages if the user specified both VID+PID and dev+bus?